tests/unit/ui/components/TestExpandableCard.js

```diff
@@ -32,24 +45,26 @@ describe(__filename, () => {
   });
 
   it('toggles when clicked', () => {
-    // We have to mount for this test because of the usage of `setState`
-    // in this component.
-    const root = mount(<ExpandableCard i18n={fakeI18n()} />);
-    const card = () => root.find('.ExpandableCard');
-    const fakeEvent = createFakeEvent();
+    const { store } = dispatchClientMetadata();
+
+    const root = render({ store });
+    const cardHeader = root.prop('header');
```

Comment:
It's not mandatory but if you check for `header` right above this line it will make a better test failure:

```js
expect(root).toHaveProp('header');
const cardHeader = root.prop('header');
// ...
```